### DISCUSSION TITLE
faq/tool/sshd/

### DISCUSSION THREAD

**Discussion by giscus** [physics-data/faq#179] [Giscus]
# faq/tool/sshd/

FAQ for Big Data in Experimental Physics

https://physics-data.meow.plus/faq/tool/sshd/

**Comment by 6yq** [comment]
如果在Windows系统中用WSL使用Ubuntu或Debian，很可能采用的是SysV init进程，
因此在使用`systemctl`命令时会报错：

```
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
```

可以考虑把启动命令改成：

```shell
sudo service ssh start
```

注意ssh和start的顺序和systemctl命令不同。设置自启动可以使用`sysv-rc-conf`程序：

```shell
sudo apt-get install sysv-rc-conf
sudo sysv-rc-conf ssh on
```

重新登陆Linux以后可以查询是否自启动：

```shell
sudo service ssh status
```

**Reply by 6yq** [reply]
p.s.
1. 代码块换行以后好像没有对齐
2. 第三个代码块在发送的时候手抖删掉了换行，应该是这样的：

```objc
sudo apt-get install sysv-rc-conf
sudo sysv-rc-conf ssh on
```

**Reply by jiegec** [reply]
感谢这位同学的补充！我已经修复了上面的换行问题
